pkg/operator/feature_gates.go

```diff
@@ -61,3 +61,7 @@ func mapToString(m map[string]bool) string {
 	}
 	return strings.Join(s, ",")
 }
+
+func HasFeatureGate(feat string) bool {
+	return defaultFeatureGates[feat]
+}
```

Comment:
This isn't necessary, see comments above :)

Reply:
I don't have a hard opinion on that, but I do think it's more idiomatic, I'd would choose one I'll choose the HasFeatureGate 😅 

But as you guys prefer.

Reply:
Thanks. Got it.  Please re-check